docs/plugins/index.rst

```diff
@@ -66,6 +66,7 @@ like this::
    inline
    ipfs
    keyfinder
+   kodinfo
```

Comment:
Done! Added the description down below in the interoperability section. I hope it's the correct one.